source/frontend/src/components/maps/hooks/useSupercluster.ts

```diff
@@ -46,7 +56,7 @@ const useSupercluster = <
      * greater than 100 (avoid perfomrance issues when rendering all points)
      */
     if (bounds && (options?.enableClustering || points.length > 100)) {
-      setClusters(superclusterRef.current.getClusters(bounds, zoomInt));
```

Comment:
Due to useEffects and other prop changes, this method would be called 3-4 times per user interaction prior to this change.